azure/macos/job.yml

```diff
@@ -85,6 +85,11 @@ jobs:
           configurationName: ${{ parameters.configurationName }}
           runTestsName: 'OpCache'
           runTestsParameters: -d zend_extension=opcache.so -d opcache.enable_cli=1 -d opcache.protect_memory=1
+      - template: test.yml
+        parameters:
+          configurationName: ${{ parameters.configurationName }}
+          runTestsName: 'OpCache shm'
+          runTestsParameters: -d zend_extension=opcache.so -d opcache.enable_cli=1 -d opcache.protect_memory=1 --file-cache-use
```

Comment:
I don't get what this is supposed to be testing.

Reply:
Shouldn't there be `-d opcache.preferred_memory_model=shm`? And it seems to me that `--file-cache-use` doesn't belong here.

Reply:
thx you both for your inputs.

Reply:
I'm still not sure whether this is testing something meaningful. If the "shm" model doesn't work, won't this still fall back to a different memory model that does work?

Reply:
Oh, right! I think to verify that the desired model has been chosen, we could add a test that is skipped unless `opcache.preferred_memory_model=shm` and then use phpinfo() to find which memory model is used:
````.php
ob_start();
phpinfo();
$info = ob_get_clean();
if (preg_match('/^Shared memory model => (.*)/um', $info, $m)) {
    var_dump($m[1]);
}
````

Reply:
thanks @cmb69 and above all @nikic in fact it does not work.